draft-ietf-ppm-dap.md

```diff
@@ -1912,7 +1913,7 @@ The Leader proceeds as follows with each report:
    job.
 
 Since VDAF preparation completes in a constant number of rounds, it will never
-be the case that some reports are completed and others are not.
+be the case that preparation is complete for some reports but not others.
```

Comment:
I think this will conflict with #702. Should we resolve and merge that PR before this one?